src/Form/AgeCheckerMappingAdminForm.php

```diff
@@ -0,0 +1,228 @@
+        '#description' => t('Please enter the remember me text.'),
+      );
+
+      // Age checker submit button text.
+      $form[$language . '_mapping']['age_checker_' . $language . '_button_text'] = array(
+        '#type' => 'textfield',
+        '#title' => t('Label of submit button'),
+        '#maxlength' => 50,
+        '#required' => TRUE,
+        '#default_value' => $config->get('age_checker_' . $language . '_button_text') ? $config->get('age_checker_' . $language . '_button_text') : $default_settings->get('age_checker_button_text'),
+        '#description' => t('Please enter submit button text.'),
+      );
+
+      // Configuration for adding Footer Links.
+      $form[$language . '_mapping']['age_checker_' . $language . '_footer_links'] = array(
+        '#type' => 'textarea',
+        '#title' => t('Footer text and link'),
+        '#required' => FALSE,
+        '#maxlength' => 255,
+        '#default_value' => $config->get('age_checker_' . $language . '_footer_links', ''),
+        '#description' => t('Please enter footer links like Privacy policy, Terms of use, cookie policy in key|value format for e.g - Privacy Policy | http://site.link'),
+      );
+
+      // Copy right text.
+      $copyright_text = $config->get('age_checker_' . $language . '_copyright', array('value' => '', 'format' => NULL));
+      $form[$language . '_mapping']['age_checker_' . $language . '_copyright'] = array(
+        '#type' => 'text_format',
+        '#title' => t('Copyright text'),
+        '#description' => t('This field is used for the footer copyright text.'),
+        '#rows' => 6,
+        '#default_value' => isset($copyright_text['value']) ? $copyright_text['value'] : '',
+        '#description' => t('Copyright © 2015 Brand name. All rights reserved'),
+        '#format' => $copyright_text['format'],
+      );
+    }
+
+    return parent::buildForm($form, $form_state);
+  }
+
+  /**
+   * Implements hook_form_submit().
+   *
+   * @param array $form
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    $config = $this->config('age_checker_mapping.settings');
+    $languages = \Drupal::state()->get('age_checker_language', 'Please provide values');
+    $languages = explode("\n", $languages);
+    foreach ($languages as $language) {
+      $language = explode('|', $language);
+      $language = array_map('trim', $language);
+      $language = $language[0];
+
+      \Drupal::state()->set('age_checker_' . $language . '_country_list', $form_state->getValues()['age_checker_' . $language . '_country_list']);
+      \Drupal::state()->set('age_checker_' . $language . '_age_gate_header', $form_state->getValues()['age_checker_' . $language . '_age_gate_header']);
```

Comment:
Avoid using $form_state->getValues() multiple times, instead go for storing values in a variable.